tower-http/src/services/fs/mod.rs

```diff
@@ -138,6 +139,16 @@ where
             reader: ReaderStream::with_capacity(read, capacity),
         }
     }
+
+    fn with_capacity_limited(
+        read: T,
+        capacity: usize,
+        max_read_bytes: u64,
+    ) -> AsyncReadBody<Take<T>> {
+        AsyncReadBody {
+            reader: ReaderStream::with_capacity(read.take(max_read_bytes as u64), capacity),
```

Comment:
`max_read_bytes` seems to already be a `u64`

```suggestion
            reader: ReaderStream::with_capacity(read.take(max_read_bytes), capacity),
```

Reply:
Yeah, missed removing that when migrating from usize -> u64, nice catch